internal/tensorzero-auth/src/postgres/mod.rs

```diff
@@ -77,12 +78,15 @@ pub enum AuthResult {
     MissingKey,
 }
 
-#[derive(sqlx::FromRow, Debug, PartialEq, Eq, Clone)]
+#[derive(sqlx::FromRow, Debug, PartialEq, Eq, Clone, Serialize)]
+#[cfg_attr(test, derive(ts_rs::TS))]
+#[cfg_attr(test, ts(export))]
```

Comment:
Let's add `ts(optional_fields)` here so the generated bindings have `description?: string` instead of `string | null` etc. We should also generally `#[serde(skip_serializing_if = "Option::is_none")]` for these?